package.json

```diff
@@ -56,6 +56,14 @@
     "jest-watch-typeahead": "^1.0.0",
     "npm-run-all": "^4.1.5",
     "prettier": "^2.5.0",
+    "@rollup/plugin-babel": "^5.3.1",
+    "@rollup/plugin-commonjs": "^21.1.0",
+    "@rollup/plugin-image": "^2.0.5",
+    "@rollup/plugin-json": "^4.1.0",
+    "@rollup/plugin-node-resolve": "^13.2.1",
+    "@rollup/plugin-replace": "^2.3.3",
+    "@rollup/plugin-virtual": "^2.0.3",
+    "@rollup/pluginutils": "^4.1.0",
     "stylelint": "^14.1.0",
     "turbo": "^1.2.8",
     "typescript": ">=3.0.0"
```

Comment:
Do we need to bump this version of `typescript`? Polaris was previously on `~4.3.5`.

Reply:
Yep we do. `yarn why typescript` shows multiple versions of typescript installed. We should remove mentions of typescript on a package level and just have one definition at the root. This could be done outside of this PR, possibly in #5962, that pulls other packages up to the root

Reply:
Yes. I agree a good follow up.